source/style-guide/information-types/concept.txt

```diff
@@ -0,0 +1,101 @@
+======================
+Concept Page Prototype
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/info-typing/fact-general-guidance.rst
+
+A concept page helps a user learn about a concept or new feature. The page intends to 
+answer the following questions:
+
+- "What is this?"
+
+- "Why do I care?"
+
+- "How does this work?"
+
+The audience includes users who have never used a particular feature or
+command.
+
+The title is typically a noun or noun phrase. 
+
+A concept page begins with a brief introduction that summarizes its
+contents, often called a short description. The short description
+explains a little about the concept and why the audience should care
+about it.
+
+Command Syntax
+--------------
+
+Optional.
+
+If it's appropriate, you may include a code-block with a copyable
+prototype to familiarize users with command syntax.
+
+.. code-block:: javascript
+
+   db.collection.<method>( {
+      /* command fields */
+   } )
+
+After the code block, provide a link to the corresponding
+:ref:`reference <meta-reference-prototype>` or :ref:`task
+<meta-task-prototype>` page to provide readers with complete examples
+and behavior details. When possible, single-source code examples and
+prototypes across pages.
+
+.. note::
+
+   For concept pages that have multiple linked tasks, the code example
+   should show the most common task we expect users to look for.
+
+Use Cases
+---------
+
+This section includes a collection of use cases to explain "What do I do
+with this thing?" In general, use cases are short descriptions about how
+to use a feature. They set expectations for when and how you might use a
+feature and what differentiates this feature from other options.  
+
+Behavior
+--------
+
+This section includes information you need to know before using a
+feature. It can be as long or short as appropriate. Some examples of
+technical details include:
+
+- Performance considerations and behavior alerts
+
+- Support for a feature (for example, sharded cluster support or
+  Versioned API compatibility)
+
+- Version support 
+
+Get Started
+-----------
+
+Link to the most common task pages related to this concept. Linking to
+all task pages isn't necessary here. The goal of this section is to make
+sure users can easily find common related tasks.
+
+- Link one
+- Link two
+- Link three
+
+Details
```

Comment:
Question: Should this section come before Get Started? It seems like we might want to present readers with all the information before encouraging them to try out the feature.

Reply:
The idea is that this section would be lower-level details, such as technical implementation. Generally, the content in this section should not influence whether or not a user uses a feature, but instead provides some additional technical information. The Behavior section is where we'd put information that might affect whether or not a user wants to use a feature, such as compatibility and version support.

I will update this section to try to clarify this point.